src/platform_impl/windows/mod.rs

```diff
@@ -89,12 +94,29 @@ unsafe impl Sync for WindowId {}
 
 impl WindowId {
     pub const unsafe fn dummy() -> Self {
-        use std::ptr::null_mut;
-
-        WindowId(null_mut())
+        WindowId(0)
     }
 }
 
+#[allow(non_snake_case)]
+#[inline(always)]
+unsafe fn GetWindowLongPtrW(hwnd: HWND, nindex: WINDOW_LONG_PTR_INDEX) -> isize {
```

Comment:
Can we name this something else? It seems like one could easily mistake this for the built-in `GetWindowLongPtrW`

Reply:
Renamed it. Yeah it was pretty confusing 😄